android/app/src/main/assets/microlocations

```diff
@@ -1,122 +1,10 @@
 [
```

Comment:
Don't remove sample data. Please change attributes names using script you wrote. Sample should contain more data.